pages/common/pip.md

```diff
@@ -3,14 +3,10 @@
 > Python package manager.
 > More information: <https://pip.pypa.io>.
 
-- Install a package:
+- Install a package (see `tldr pip install` for more install examples):
```

Comment:
I suggest avoiding the `tldr ` here, and keeping it to just the command + subcommand.
```suggestion
- Install a package (see `pip install` for more install examples):
```